### ISSUE TITLE
Outil de génération de graphes

### ISSUE THREAD

**Issue by lemaitre** [lemaitre/thesis#5]
# But
Développer un outil pour générer automatiquement des graphes à partir des benchmarks.

# bibliothèques
- Matplotlib: python
- Root: C++
- gnuplot: shell

# Stockage des benchs
- csv
- sqlite?
- rootfile?

**Comment by bcouturi** [comment]
je pense qu'il vaut mieux garder quelque chose de simple pour le stockage.
Dans tous les cas un script pour reconvertir vers CSV est necessaire !

**Comment by lemaitre** [comment]
Je suis d'accord pour quelque chose de simple, mais le csv montre vite ses limites quand on a plusieurs paramètres.
Pour l'instant, j'avais fait avec plusieurs csv, mais quand on doit fusionner des centaines de csv pour en sortir un nouveau qui servira pour le graphe, ça commence à devenir compliqué.
C'est pourquoi j'avais pensé à sqlite ou root.
